### ISSUE TITLE
Improve system call to create and remove the d2_aperture file

### ISSUE THREAD

**Issue by chernals** [lhcopt/hllhc15#4]
I know this is totally minor, however, it would prevent unwanted warnings which can confuse the faint-hearted:

In `aperture_upgrade_MS.madx` could we change this:

```
system,"rm d2_aperture";assign, echo="d2_aperture";
exec,write_OCTAGON(xx1,yy1,xx2,yy2);
assign, echo=terminal; 
```

by

```
system,"if [ -f d2_aperture ]; then rm d2_aperture; fi";assign, echo="d2_aperture";
exec,write_OCTAGON(xx1,yy1,xx2,yy2);
assign, echo=terminal; 
```

**Comment by rdemaria** [comment]
ok for me, please make a PR... At some point the d2_aperture file could disappear as MAD-X has now the octagon shape...

**Comment by rdemaria** [comment]
fixed by #5
